### PR TITLE
typo in README

### PR DESCRIPTION
**Pull request by Yakushima**

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -45,7 +45,7 @@ group :development do
   gem 'spring'
 end
 
-gem 'mysql2'
+# gem 'mysql2'
```
Comment (StanfordCDT) [inline Gemfile:48]: We use MySQL for this project. Please don't change this file.

File: Gemfile.lock [hunk 2]
```diff
@@ -96,7 +96,6 @@ GEM
     mini_portile2 (2.1.0)
     minitest (5.9.0)
     multi_json (1.12.1)
-    mysql2 (0.4.4)
```
Comment (StanfordCDT) [inline Gemfile.lock:99]: We use MySQL for this project. Please don't change this file.

File: db/schema.rb [hunk 3]
```diff
@@ -14,203 +14,203 @@
...
+    t.text     "user_agent"
+    t.string   "stage"
+    t.boolean  "void",                         default: false, null: false
+    t.text     "data"
+    t.datetime "created_at",                                   null: false
+    t.datetime "updated_at",                                   null: false
   end
 
-  add_index "voters", ["election_id", "authentication_method", "authentication_id"], name: "index_voters_on_election_id_and_authentication", unique: true, using: :btree
+  add_index "voters", ["election_id", "authentication_method", "authentication_id"], name: "index_voters_on_election_id_and_authentication", unique: true
```
Comment (StanfordCDT) [inline db/schema.rb:214]: Please don't change this file. These attributes are for MySQL.

**Review by StanfordCDT — COMMENTED** [review]
Thank you for adding a Swahili locale!

Please don't add /pb and /pb_test to the repository. These are SQLite files.

Please don't add /config/database.yml and /config/secrets.yml to the repository. These are local config files. They should have been ignored by `.gitignore`.

Feel free to add your name to the AUTHORS file.

**Comment by lgelauff** [comment]
I changed the typo part myself. I'm closing this PR as the rest of the changes seems outdated by now.

**Pull request closed without merging**